files/image_config/misc/docker-wait-any

```diff
@@ -11,13 +11,12 @@
     stopped running. Here, we spawn multiple threads and wait on one
     container per thread. If any of the threads exit, the entire
     application will exit.
-    NOTE: This script is written against docker-py version 1.6.0. Newer
-    versions of docker-py have a different API.
+    NOTE: This script is written against docker 4.1.0.
```

Comment:
>docker [](start = 41, length = 6)

docker-py?

Reply:
no. the project is named as docker now. check out this one. https://pypi.org/project/docker/